lib/sdf-v1-routes-session/src/auth_connect.rs

```diff
@@ -281,7 +283,7 @@ pub async fn auth_connect(
 
     let ctx = builder.build_default(request_ulid).await?;
 
-    let (user, workspace) = find_or_create_user_and_workspace(
+    let (ctx, user, workspace) = find_or_create_user_and_workspace(
```

Comment:
what's going on with the dal context here?

Reply:
ok i understand it, it's workable!

Reply:
When I tried just making the passed argument a mutable ref, it broke because the &mut does not implement the SiDbCtx trait that an inner function needs. So i just let it swallow a ctx and return the modified one. Clunky but functional